<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Computer Vision for Microscopy Image Analysis 2021</title>
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/speakers.css">
</head>

<body>
    <div class="headBox">
   <!--     <div class="imgBox">
            <img class="logoBox" src="img/images/Picture2.png" alt="">

        </div>  -->

        <div class="navBox">
            <ul>
                <li>
                    <a href="./new.html"><i><strong class="new">New</strong></i></a></li>
                <li>
                    <a href="index.html"><strong>Welcome</strong></a></li>
                <li>
                    <a href="organizers.html"><strong>Organizers</strong></a></li>
                <li>
                    <a href="speakers.html"><strong>Invited Speakers</strong></a></li>
		<li>
                    <a href="program.html"><strong>Program</strong></a></li>
		<li>
                    <a href="registration.html"><strong>Registration</strong></a></li>
                <li>
                    <a href="venue.html"><strong>Attend</strong></a></li>
                <li>
                    <a href="sponsor.html"><strong>Sponsors</strong></a></li>
		<li>
                    <a href="jobs.html"><strong>Jobs</strong></a></li>
		<li>
                    <a href="comittee.html"><strong>Program Committee</strong></a></li>
                <li>
                    <a href="callForPaper.html"><strong>Call for Papers</strong></a></li>
                <li>
                    <a href="dates.html"><strong>Important Dates</strong></a></li>
                <li>
                    <a href="submission.html"><strong>Submission</strong></a></li>
                <li>
                    <a href="accepted.html"><strong>Accepted Papers</strong></a></li>
		<li>
                    <a href="spotlight.html"><strong>Works-in-Progress</strong></a></li>
		<li>
                    <a href="https://motchallenge.net/data/CTMC-v1/" target="_blank"><strong>CTMC Challenge</strong></a></li>

                <li>
                    <a href="contact.html"><strong>Contact</strong></a></li>
                <li>
                    <a href="pastcvmi.html"><strong>Past CVMIs</strong></a></li>
            </ul>
        </div>
    </div>

    <div class="contentBox">
        <div class="headImgBox">
            <img class="headImg" src="img/images/Picture23.png" alt="">
        </div>

        <div class="titleBox">
            <p style="font-size: 120%;font-weight: 700;">Invited Speakers</p>
        </div>
        <hr style="height:1px;border:none;border-top:1px solid #555555;">
      <div class="detailBox detailBoxSpeakers">

	  <div>
          <p>  <strong>Talk Title:</strong> Information in images for drug discovery: image-based profiling </p>
             <strong>Start Time: </strong><span>9:40 AM PDT</span><br>		
		<strong>Speaker:</strong>
                <span>Inti Zlobec, Professor, University of Bern. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Inti.png" width="200" height="225">
		</center></p>
		<strong> Abstract:</strong> Cell images contain a vast amount of quantifiable information about the status of the cell: for example, whether it is diseased, whether it is responding to a drug treatment, or whether a pathway has been disrupted by a genetic mutation. We extract hundreds of features of cells from images. Just like transcriptional profiling, the similarities and differences in the patterns of extracted features reveal connections among diseases, drugs, and genes. Improving this pipeline is an active area of research, from feature extraction to batch correction to quality control to assessing similarities.

We are harvesting similarities in image-based profiles to identify, at a single-cell level, how diseases, drugs, and genes affect cells, which can uncover small molecules’ mechanism of action, discover gene functions, predict assay outcomes, discover disease-associated phenotypes, identify the functional impact of disease-associated alleles, and find novel therapeutic candidates. As part of the JUMP-Cell Painting Consortium (Joint Undertaking for Morphological Profiling-Cell Painting) we are aiming to establish experimental and computational best practices for image-based profiling (https://jump-cellpainting.broadinstitute.org/results) and produce the world’s largest public Cell Painting gene/compound image resource, with 140,000 perturbations in five replicates, to be released November 2022. With these data and new technologies like Pooled Cell Painting and variants of the assay like LipocyteProfiler and CardioProfiler, we hope to bring drug discovery-accelerating applications to practice.
		</p>	
		<strong>Speaker Bio:</strong>
                <span>Inti Zlobec holds the position of Professor (Extraordinarius) of Digital Pathology at the Institute of Pathology, University of Bern, Switzerland. She graduated with a PhD degree in Experimental Pathology, from McGill University, Montreal, Canada in 2007 before completing a post-doctoral fellowship at the Institute of Pathology, University Hospital Basel, where she conducted tissue-based research in the field of colorectal cancer using biostatistical models. After habilitating in 2010, she received a position at the Institute of Pathology, University of Bern, where she established and led the Translational Research Unit (TRU) and later the Tissue Bank Bern (TBB). Inti Zlobec became Associate Professor in 2014. Now, she leads an inter-disciplinary research group of students and researchers using artificial intelligence and machine learning as tools to study pathology images along with other data types to discover and validate novel prognostic and predictive biomarkers for colorectal cancer patients. Inti Zlobec is a member of the Executive Team of the Center for Artificial Intelligence in Medicine (CAIM) of the University of Bern, Co-Founder and President of the Swiss Consortium for Digital Pathology (SDiPath) and Chair of the European Society of Pathology (ESP) Working Group IT.
                </span>
            </div> 

<P>

</p>

       <hr style="height:1px;border:none;border-top:1px solid #555555;" />   
            <div>

<!--		<p>  <strong>Talk Title:</strong> Machine learning challenges in spatial single cell omics analysis </p>	-->
	 		<strong>Start Time: </strong><span>8:40 AM PDT</span><br>	
                <strong>Speaker:</strong>
                <span>Paula A. Marin Zapata, Data Scientist, Bayer, Berlin. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Paula.jpeg" width="200" height="200">
		</center></p>
	<!--	<strong> Abstract:</strong> Methods for profiling RNA and protein expression in a spatially resolved manner are rapidly evolving, making it possible to comprehensively characterize cells and tissues in health and disease. The resulting large-scale and complex multimodal data sets raise interesting computational and machine learning challenges, from QC & storage to actual analysis. For instance what is the best description of a local cell neighborhood, how do we find such interesting ones and how are these differential across disease or other perturbations? And how can they be chained together to build a spatial human cell atlas?

Here, I will present approaches from the lab touching upon a few of these points. In particular, I will show how our recent toolbox Squidpy and the related SpatialData format support standard steps in analysis and visualization of spatial molecular data. I will then discuss recent approaches towards multimodal classification, learning cell cell communication and extension towards morphometric representations under perturbations using generative models.
            </p>  	-->
		<strong>Speaker Bio:</strong>
                <span>Paula is a data scientist from Bayer AG in Berlin. She obtained a PhD in biology from the German Cancer Research Center DKFZ, a MSc in Applied Mathematics from Eindhoven University of Technology, and a BSc in Biological Engineering from the National University of Colombia in Medellin. In 2017, she joined Bayer as a postdoctoral fellow, where she developed deep learning methods for phenotypic profiling in plant sciences and cellular images. Since 2019, she works at the Machine Learning Research group from Bayer R&D, where she focuses on image analysis applications to drug discovery.

            <br>
		</span>  
	      </div>

      <!--           <hr style="height:1px;border:none;border-top:1px solid #555555;" />
	      <div>

		<p>  <strong>Talk Title:</strong> AI for breast cancer diagnostics 2.0</p>
                 <strong>Start Time: </strong><span>9:20 AM PDT</span><br>
		<strong>Speaker:</strong>
                <span>Jeroen van der Laak, Professor, Radboud University Medical Center. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Jeroen.jpg" width="190" height="200">
		</center></p>
		<strong> Abstract:</strong> Deep learning is a state-of-the-art pattern recognition technique that has proven extremely powerful for the analysis of digitized histopathological slides. In our work, we studied the use of deep learning to assess a range of breast-cancer related tissue features: presence of lymph node metastases, extend of lymphatic infiltrate within tumors, and the components of tumor grading. It was shown that DL enables reproducible, quantitative tumor feature extraction, showing a good correlation with pathologists’ scores and with patient outcome. Our current research involves larger scale validation with pathologists, to study the added value of the developed algorithms in routine practice, in terms of efficiency and diagnostic accuracy. Such studies will be required for certification but are still mostly lacking in our field, making it difficult to assess the true potential of deep learning for pathology diagnostics.
		</p>		
<strong>Speaker Bio:</strong>
                <span>Jeroen van der Laak is professor in Computational Pathology and principle investigator at the Department of Pathology of Radboud University Medical Center in Nijmegen, The Netherlands and guest professor at the Center for Medical Image Science and Visualization (CMIV) in Linköping, Sweden. His research group investigates the use of deep learning-based whole-slide image analysis for different applications; improvement of routine pathology diagnostics, objective quantification of immunohistochemical markers, and study of novel imaging biomarkers for prognostics. Jeroen has an MSc in Computer science and acquired his PhD from Radboud University in Nijmegen. He co-authored over 125 peer-reviewed publications and is a member of the editorial boards of Modern Pathology, Laboratory Investigation and the Journal of Pathology Informatics. He is chair of the taskforce ‘AI in Pathology’ of the European Society of Pathology, member of the board of directors of the Digital Pathology Association and organizer of the Computational Pathology Symposium at the European Congress of Pathology. He coordinated the CAMELYON grand challenges in 2016 and 2017. Jeroen van der Laak acquired research grants from the European Union and the Dutch Cancer Society, among others. Jeroen is coordinator of the Bigpicture consortium and USCAP Nathan Kaufman lecture laureate.</span>
            </div>

            <hr style="height:1px;border:none;border-top:1px solid #555555;" />
            <div>

           <p>  	<strong>Talk Title:</strong> How Can Humans Learn from AI</p>
            	<strong>Start Time: </strong><span>11:15 AM PDT</span><br>	
                <p><strong>Speaker:</strong> 	
                <span>Po-Hsuan Cameron Chen, Ph.D., Need.</span><strong></strong> </p>
		<p><center>
		<img class="img-responsive" alt="" src="./img/images/Cameron.jpg" width="200" height="200">
		</center></p>
          	<strong> Abstract:</strong> In traditional ML, models learn from hand-engineered features informed by existing domain knowledge. More recently, in the deep learning era, combining large-scale model architectures, compute, and datasets has enabled learning directly from raw data, often at the expense of human interpretability. In this talk, I'll discuss using deep learning to predict patient outcomes with interpretability methods to extract new knowledge that humans could learn and apply. This process is a natural next step in the evolution of applying ML to problems in medicine and science, moving from the use of ML to distill existing human knowledge to people using ML as a tool for knowledge discovery.           
            	</p>	
                <strong>Speaker Bio:</strong>
                <span>Cameron Chen is the Head of AI and Data Science at Need, a tech startup developing the world's first cancer protection system. Before that, he was a staff software engineer and tech lead manager at Google Research and Google Health. His primary research interests lie at the intersection of machine learning and healthcare. His research has been covered by various media outlets, including the New York Times, Forbes, Engadget, etc. He received his PhD degree in Electrical Engineering and Neuroscience from Princeton University and his BS from National Taiwan University. Cameron was also a recipient of the Google PhD Fellowship.</span>
            </div>

            <hr style="height:1px;border:none;border-top:1px solid #555555;" />
            <div>

	       <p>  <strong>Talk Title:</strong> Decoding hidden signal from neurodegenerative drug discovery high-content screens  </p>
               <strong>Start Time: </strong><span>1:00 PM PDT</span><br>	
		<strong>Speaker:</strong>
                <span>Michael J Keiser, Associated Professor, UC San Francisco. </span> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Keiser.jpg" width="150" height="220">
		</center></p>
		<p>
        	<strong> Abstract:</strong> Alzheimer's disease is a complex and recalcitrant condition that has largely evaded traditional molecular drug discovery approaches. Phenotypic drug discovery using high-content cellular models with unbiased small molecule screening is promising but faces obstacles from subtle signal, artifacts, and non-specific visual markers. We propose two deep learning-based methods to overcome these challenges in large-scale cellular screens. First, we develop deep neural networks to generate missing fluorescence channel images from an Alzheimer's disease high-content screen (HCS), enabling the identification and prospective validation of overlooked but active small molecules. This is a unique application of generative images in drug discovery. Second, we introduce a learned biological landscape using deep metric learning to organize drug-like molecules by live-cell HCS images. Metric learning outperforms conventional image scoring and reveals previously hidden molecules that push diseased cells toward a healthy state as effectively as positive control compounds. These results indicate that a wealth of actionable biological information lies untapped but readily available in HCS datasets.
            	</p>
		<strong>Speaker Bio:</strong>
                <span>As a National Science Foundation Fellow, Dr. Keiser earned a PhD in bioinformatics in 2009 from UCSF, where he developed techniques, such as the Similarity Ensemble Approach (SEA), to relate drugs and proteins based on the statistical similarity of their ligands. Dr. Keiser also holds BSc, BA, and MA degrees from Stanford University. He subsequently cofounded a startup that brings these methods to pharmaceutical companies and to the US FDA. Dr. Keiser joined the faculty at UCSF in the Department of Pharmaceutical Chemistry and the IND in 2014, with a joint appointment in the Department of Bioengineering and Therapeutic Sciences. His lab investigates forward polypharmacology for complex diseases and the prediction of drug off-target activities.
The Keiser lab combines machine learning and chemical biology methods to investigate how small molecules perturb entire protein networks to achieve their therapeutic effects. In classical pharmacology, each drug was thought to strike a single note (in other words, "one drug hits one target to treat one disease"). However, it has been discovered that some drugs strike entire "chords" of targets at once, and this can be essential to their action. The Keiser group is tracing out this molecular music, not only in terms of new and useful therapeutic chords to treat neurodegenerative diseases, but also to identify the jarring notes that some drugs might unintentionally hit when they induce side effects.   </span>
            </div>

         <hr style="height:1px;border:none;border-top:1px solid #555555;" />
	    <div>
       	<p>  <strong>Talk Title:</strong> Multimodal Computational Pathology</p>
		<strong>Start Time: </strong><span>1:40 PM PDT</span><br>	
        	<p> <strong>Speaker:</strong>
		<span>Faisal Mahmood, Associate Professor, Harvard Medical School.</span></p>
        	<p><center>
		<img class="img-responsive" alt="" src="./img/images/Faisal.jpg"  width="195" height="200">
		</center></p>
        
       		 <p>
    		<strong> Abstract:</strong> 
         	Advances in digital pathology and artificial intelligence have presented the potential to build assistive tools for objective diagnosis, prognosis and therapeutic-response and resistance prediction. In this talk we will discuss our work on: 1) Data-efficient methods for weakly-supervised whole slide classification with examples in cancer diagnosis and subtyping (Nature BME, 2021), and allograft rejection (Nature Medicine, 2022) 2) Harnessing weakly-supervised, fast and data-efficient WSI classification for identifying origins for cancers of unknown primary (Nature, 2021). 3) Discovering integrative histology-genomic prognostic markers via interpretable multimodal deep learning (Cancer Cell, 2022; IEEE TMI, 2020; ICCV, 2021). 4) Self-supervised deep learning for pathology and image retrieval (CVPR, 2022; Nature BME, 2022). 5) Integrating vision and language for computational pathology (CVPR, 2023) 6) Deploying weakly supervised models in low resource settings without slide scanners, network connections, computational resources, and expensive microscopes. 7) Bias and fairness in computational pathology algorithms.
          	</p>		
          
		<strong>Speaker Bio: </strong>
		<span>Dr. Mahmood is an Associate Professor of Pathology at Harvard Medical School and the Division of Computational Pathology at the Brigham and Women's Hospital. He received his Ph.D. in Biomedical Imaging from the Okinawa Institute of Science and Technology, Japan and was a postdoctoral fellow at the department of biomedical engineering at Johns Hopkins University. His research interests include pathology image analysis, morphological feature, and biomarker discovery using data fusion and multimodal analysis. Dr. Mahmood is a full member of the Dana-Farber Cancer Institute / Harvard Cancer Center; an Associate Member of the Broad Institute of Harvard and MIT, and a member of the Harvard Bioinformatics and Integrative Genomics (BIG) faculty.</span>


    	    <hr style="height:1px;border:none;border-top:1px solid #555555;" />
	    <div>

           <p>  <strong>Talk Title:</strong> PLIP: Leveraging medical Twitter to build a visual–language foundation model for pathology AI </p>
			<strong>Start Time: </strong><span>3:30 PM PDT</span><br>		
		<strong>Speaker:</strong>
                <span>James Zou, Assistant Professor, Stanford University. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/JamesZou.jfif" width="210" height="210">
		</center></p>
		<strong> Abstract:</strong>The lack of annotated publicly available medical images is a major barrier for innovations. At the same time, many de-identified images and much knowledge are shared by clinicians on public forums such as medical Twitter. Here we harness these crowd platforms to curate OpenPath, a large dataset of 208,414 pathology images paired with natural language descriptions. This is the largest public dataset for pathology images annotated with natural text. We demonstrate the value of this resource by developing PLIP, a multimodal AI with both image and text understanding, which is trained on OpenPath. PLIP achieves state-of-the-art zero-shot and transfer learning performances for classifying new pathology images across diverse tasks. Moreover, PLIP enables users to retrieve similar cases by either image or natural language search, greatly facilitating knowledge sharing. Our approach demonstrates that publicly shared medical information is a tremendous resource that can be harnessed to advance biomedical AI.</p>
		
		<strong>Speaker Bio:</strong>
                <span>James Zou is an assistant professor of biomedical data science and, by courtesy, of CS and EE at Stanford University. Professor Zou develops novel machine and deep learning algorithms that have strong statistical guarantees; several of his methods are currently being used by biotech companies. He also works on questions important for the broader impacts of AI—e.g. interpretations, robustness, transparency—and on biotech and health applications. He has received several best paper awards, a Google Faculty Award, a Tencent AI award and is a Chan-Zuckerberg Investigator. He is also the faculty director of Stanford AI for Health program and is a member of the Stanford AI Lab. https://www.james-zou.com/.
                </span>
            </div>

    	    <hr style="height:1px;border:none;border-top:1px solid #555555;" />
	    <div>

           <p>  <strong>Talk Title:</strong> Point-and-click: using microscopy images to guide spatial next generation sequencing measurements </p>
			<strong>Start Time: </strong><span>4:00 PM PDT</span><br>		
		<strong>Speaker:</strong>
                <span>Jocelyn Kishi, PhD, CEO & Co-Founder, Stealth TechBio Startup. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Josie.jpg" width="210" height="210">
		</center></p>
		<strong> Abstract:</strong> By using microscopy images of cells to guide multiplexed spatial indexing of sequencing reads, Light-Seq allows combined imaging and spatially resolved next generation sequencing (NGS) measurements to be captured from fixed biological samples. This is achieved through the combination of spatially targeted, rapid photocrosslinking of DNA barcodes onto complementary DNAs in situ with a one-step DNA stitching reaction to create pooled, spatially indexed sequencing libraries. Selection of cells can be done manually by pointing and clicking on the regions of interest (ROIs) to be sequenced, or automatically through the use of computer vision for cell type identification and segmentation. This foundational capability opens up a broad range of applications for multi-omic analysis unifying microscopy and NGS measurements from intact biological samples.</p>
		
		<strong>Speaker Bio:</strong>
                <span>Dr. Kishi’s research focuses on developing new methods for reading and writing DNA sequences, DNA computing and molecular robotics, and DNA-based imaging assays. My background in software engineering has allowed me to work on projects at the intersection of Computer Science and DNA Nanotechnology.
                </span>
            </div>



    	    <hr style="height:1px;border:none;border-top:1px solid #555555;" />
	    <div>

           <p>  <strong>Talk Title:</strong> Enhancing SAM's Biomedical Image Analysis through Prompt-based Learning </p>
			<strong>Start Time: </strong><span>5:00 PM PDT</span><br>		
		<strong>Speaker:</strong>
                <span>Dong Xu, Curators’ Distinguished Professor, Uuniversity of Missouri, Columbia. </span><strong></strong> <br>
                <p><center>
		<img class="img-responsive" alt="" src="./img/images/Dong.jpg" width="160" height="210">
		</center></p>
		<strong> Abstract:</strong> The Segment Anything Model (SAM), a foundational model trained on an extensive collection of images, presents many opportunities for diverse applications. For instance, we employed SAM in our biological pathway curation pipeline that synergizes image understanding and text mining techniques for deciphering gene relationships. SAM has proven highly efficient in recognizing pathway entities and their interconnections. However, SAM does not work well when applied to low-contrastive images directly. To counter this, we investigated prompt-based learning with SAM, specifically for identifying proteins from cryo-Electron Microscopy (cryo-EM) images. We trained a U-Net-based filter to adapt these grayscale cryo-EM images into RGB images suitable for SAM's input. We also trained continuous prompts and achieved state-of-the-art (SOTA) performance, even with a limited quantity of labeled data. The outcomes of our studies underscore the potential utilities of prompt-based learning on SAM for efficient biomedical image analyses.</p>
		
		<strong>Speaker Bio:</strong>
                <span>Dong Xu is Curators’ Distinguished Professor in the Department of Electrical Engineering and Computer Science, with appointments in the Christopher S. Bond Life Sciences Center and the Informatics Institute at the University of Missouri-Columbia. He obtained his Ph.D. from the University of Illinois, Urbana-Champaign in 1995 and did two years of postdoctoral work at the US National Cancer Institute. He was a Staff Scientist at Oak Ridge National Laboratory until 2003 before joining the University of Missouri, where he served as Department Chair of Computer Science during 2007-2016 and Director of Information Technology Program during 2017-2020. Over the past 30 years, he has conducted research in many areas of computational biology and bioinformatics, including single-cell data analysis, protein structure prediction and modeling, protein post-translational modifications, protein localization prediction, computational systems biology, biological information systems, and bioinformatics applications in human, microbes, and plants. His research since 2012 has focused on the interface between bioinformatics and deep learning. He has published more than 400 papers with more than 21,000 citations and an H-index of 73 according to Google Scholar. He was elected to the rank of American Association for the Advancement of Science (AAAS) Fellow in 2015 and American Institute for Medical and Biological Engineering (AIMBE) Fellow in 2020.
                </span>
            </div>   


        </div> -->
<br>

    </div> 
</body>

</html>